templates/js/custom-copilot-rag-azure-ai-search/src/app/customSayCommand.js

```diff
@@ -0,0 +1,67 @@
+const botbuilder = require("botbuilder");
+const Utilities = require("@microsoft/teams-ai");
+
+function sayCommand(feedbackLoopEnabled = false) {
+  return async (context, _state, data) => {
+    if (!data.response?.content) {
+      return "";
+    }
+    let content = "";
+    let result = JSON.parse(data.response.content);
```

Comment:
Add error handling, e.g., if not valid json, just send as raw text

Reply:
ok, added